src/RimDev.Stuntman.AspNetCore/common/StuntmanOptions.cs

```diff
@@ -83,6 +83,16 @@ public StuntmanOptions AddUser(StuntmanUser user)
             return AddUser(user, Constants.StuntmanOptions.LocalSource);
         }
 
+        public StuntmanOptions AddUsers(IEnumerable<StuntmanUser> users) 
```

Comment:
Would it make sense to also add `public StuntmanOptions AddUsers(params StuntmanUser[] users) { }`? Is there recommended guidance for this situation?